### DISCUSSION TITLE
How to export usdz with textures?

### DISCUSSION THREAD

**Discussion by g2mXagent** [Autodesk/maya-usd#1369] [Q&A] [answered]
How to export usdz with textures?

**Comment by dgovil** [comment, marked as answer]
In the export dialog, if you give your exported file a `.usdz` extension, and if you have `Materials` export enabled, it will export a usdz file with the textures embedded inside.

**Reply by g2mXagent** [reply]
thank you.

**Comment by Ziethriel** [comment]
Can I export a USD stage as a USDZ? I don't want to lose all the references by importing the USD stage as Maya Data and then exporting.
